### PR TITLE
Blank Canvas Blocks: Add default styles for Verse block

### PR DESCRIPTION
**Pull request by danieldudzic**
Part of #3413

### CODE REVIEW

File: blank-canvas-blocks/sass/blocks/_verse.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pre.wp-block-verse {
+	padding: 0;
```
Comment (jffng) [inline blank-canvas-blocks/sass/blocks/_verse.scss]: Should this use the custom variables that you're creating above in theme.json?

File: blank-canvas-blocks/experimental-theme.json [hunk 2]
```diff
@@ -168,6 +168,14 @@
 						"lineHeight": 1.125,
 						"fontFamily": "var(--wp--preset--font-family--headings)"
 					}
+				},
+				"verse": {
+					"padding": {
```
Comment (jffng) [inline blank-canvas-blocks/experimental-theme.json:173]: Could this just be one variable instead of one each for top right bottom left?

**Comment by scruffian** [comment]
This looks like a really easy one to do in Gutenberg. I was going to do it today until my environment broke...

**Comment by danieldudzic** [comment]
> This looks like a really easy one to do in Gutenberg. I was going to do it today until my environment broke...

I've seen your PR: https://github.com/WordPress/gutenberg/pull/27579

Do you think there's a chance it will get merged soon?

File: blank-canvas-blocks/experimental-theme.json (hunk 2 above)
Comment (danieldudzic) [reply blank-canvas-blocks/experimental-theme.json:173]: I like the idea of those being separate. I used WordPress/gutenberg#27579 as a reference.

**Comment by scruffian** [comment]
I have made a much simpler version: https://github.com/WordPress/gutenberg/pull/29820/

Let's hope this one is quicker to merge...

**Comment by danieldudzic** [comment]
> I have made a much simpler version: [WordPress/gutenberg#29820](https://github.com/WordPress/gutenberg/pull/29820)
> 
> Let's hope this one is quicker to merge...

Cool - seems like your PR is going to get merged soon (if I understand the discussion correctly). I'm happy to close this for now.

**Pull request closed without merging**